lib/presentation/common/book_app_bar.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:book/styles/app_colors.dart';
+import 'package:book/styles/app_styles.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class AppBarLogReg extends StatelessWidget implements PreferredSizeWidget {
+  const AppBarLogReg({
+    super.key,
+    required this.titleText,
+  });
+
+  final String titleText;
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      automaticallyImplyLeading: false,
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.only(bottomLeft: Radius.circular(60)),
+      ),
+      bottom: PreferredSize(
+        preferredSize: Size.fromHeight(150),
```

Comment:
I noticed that if I change value of preferredSize nothing changes because it uses the value from the getter. 
@lecicdjuro, should we just put in parameter preferredSize instead of Size.fromHeight()?